src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -6,10 +6,10 @@
 import java.util.Properties;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String URL = "jdbc:mysql://localhost:3306/taxi_service_db";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "A3394124";
```

Comment:
Be bette if you return previos empty values i think. Becouse this is confidential info. 
Best practice, it`s save this into in some properties file, and read from there. But, for now, i think, it`s will be enough if you replace this by "YOUR USERNAME" and "YOUR PASSWORD"